connexion/decorators/parameter.py

```diff
@@ -124,10 +119,6 @@ def wrapper(request):
             else:
                 logger.debug("Context parameter '%s' not in function arguments", key)
 
-        # attempt to provide the request context to the function
```

Comment:
We still want to pass the context as an argument to the view function, so we can't just remove this. We need to include the context in `kwargs` using `context_kw`.